packages/create-next-app/templates/app-tw/js/app/layout.js

```diff
@@ -1,15 +1,14 @@
-import localFont from "next/font/local";
+import { Geist, Geist_Mono } from "next/font/google";
 import "./globals.css";
 
-const geistSans = localFont({
-  src: "./fonts/GeistVF.woff",
+const geistSans = Geist({
   variable: "--font-geist-sans",
-  weight: "100 900",
```

Comment:
Should we still define the weight we are using to reduce the size of the font files?

Reply:
I removed it based on the docs:

> An array of weight values if the font is not a [variable google font](https://fonts.google.com/variablefonts). It applies to next/font/google only.

x-ref: https://nextjs.org/docs/app/api-reference/components/font#weight